recipes/MobileSAM/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "mobilesam" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/ChaoningZhang/MobileSAM.git
```

Comment:
Maybe you can post an issue upstream?

Reply:
[Posted](https://github.com/ChaoningZhang/MobileSAM/issues/122), hopeful for a response